pydantic/main.py

```diff
@@ -231,7 +230,9 @@ def __init__(__pydantic_self__, **data: Any) -> None:  # type: ignore
         __pydantic_self__.__pydantic_validator__.validate_python(data, self_instance=__pydantic_self__)
 
     @classmethod
-    def __get_pydantic_core_schema__(cls, source: type[BaseModel], gen_schema: GenerateSchema) -> CoreSchema | None:
+    def __get_pydantic_core_schema__(
+        cls, __source: type[BaseModel], __handler: Callable[[Any], CoreSchema]
```

Comment:
It looks like this is the only place with the two leading `_` in the signature — was that on purpose? I understand it's meant to indicate it will be called positionally, just since it didn't have those elsewhere just wanted to make sure it was on purpose here.

Reply:
It is on purpose. Using `__` allows subclasses to name these whatever they want, including `_: Any` to tell their IDE they don't care about that parameter. If we did `source: ...` then they _have_ to call that parameter `source`.